docs/api.asciidoc

```diff
@@ -26,6 +26,18 @@ entirely.
 
 NOTE: You cannot access these APIs via the Console in Kibana.
 
+== Request Headers
+
+The following headers must be set on all non-whitelisted API endpoints 
+(See https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#saml[SAML Authentication]
+documentation for example whitelisting). 
+
+`kbn-xsrf: true`:: By default, Kibana requires the `kbn-xsrf` header set to true
```

Comment:
nit: Kibana _technically_ doesn't care what the header value is set to. It only cares that the header is present and set to some value.

Reply:
When referring to a setting in the APIs, we typically use `` around the setting. For example:

...{kib} requires the `kbn-xsrf` header set to `true` for API interactions.